lib/desired-caps.js

```diff
@@ -0,0 +1,102 @@
+import log from './logger';
+import validator from 'validate.js';
+
+validator.validators.isString = function(value) {
```

Comment:
Looks like, there is a scoping problem here, we should fork the library and fix module export so that we can create an instance of validator detached from the npm cache.